### PR TITLE
selfdestruct command

### PR DESCRIPTION
**Pull request by PallavBS**

### CODE REVIEW

**Review by fourjr — CHANGES_REQUESTED** [review]
What? This isnt just the selfdestruct command!

**Comment by KorewaKiyo** [comment]
You're low for trying to change the info.

File: cogs/info.py [hunk 1]
```diff
@@ -254,12 +254,12 @@ def __init__(self, bot):
...
         uptime = fmt.format(d=days, h=hours, m=minutes, s=seconds)
 
-        github = '[Click Here](https://github.com/verixx/selfbot.py/)'
+        github = '[Click Here](https://github.com/PallavBS/selfbot.py/)'
         server = '[Click Here](https://discord.gg/pmQSbAd)'
         website = '[selfbot-py.tk](http://selfbot-py.tk/)'
 
 
-        embed.add_field(name='Author', value='verixx#7220')
+        embed.add_field(name='Author', value='Pallav#1747')
```
Comment (KorewaKiyo) [inline cogs/info.py:262]: Fuck off with this

File: cogs/info.py [hunk 2]
```diff
@@ -254,12 +254,12 @@ def __init__(self, bot):
             fmt = '{d}d ' + fmt
         uptime = fmt.format(d=days, h=hours, m=minutes, s=seconds)
 
-        github = '[Click Here](https://github.com/verixx/selfbot.py/)'
+        github = '[Click Here](https://github.com/PallavBS/selfbot.py/)'
```
Comment (KorewaKiyo) [inline cogs/info.py:257]: Fuck off again

**Review by KorewaKiyo — COMMENTED** [review]
fuck off

**Pull request closed without merging**